spec/models/dialog_field_serializer_spec.rb

```diff
@@ -53,20 +54,39 @@
       end
 
       it "serializes the dialog_field with the correct values" do
-        expect(dialog_field_serializer.serialize(dialog_field)).to eq(expected_serialized_values.merge(
-          "resource_action" => "serialized resource action",
-          "values"          => "dynamic values"
-        ))
+        expect(dialog_field_serializer.serialize(dialog_field))
+          .to eq(expected_serialized_values.merge(
+                   "resource_action" => "serialized resource action",
+                   "values"          => "dynamic values"
+          ))
+      end
+
+      it 'returns all attributes of a dialog field' do
+        expect(dialog_field_serializer.serialize(dialog_field, true))
+          .to include(expected_serialized_values.merge(
+                        'id'              => dialog_field.id,
+                        'resource_action' => 'serialized resource action',
+                        'values'          => 'dynamic values'
+          ))
```

Comment:
I think there should be two contexts here, one for when you're expecting to get "limited" information back, and one where you're expecting to get everything. Probably something like:
```
let(:result) { dialog_field_serializer.serialize(dialog_field, all_attributes) }

context "when wanting the excluded set of attributes" do
  let(:all_attributes) { false }

  it "old test" do
    # expect(result).to eq(old stuff)
  end
end

context "when wanting all of the attributes" do
  let(:all_attributes) { true }

  it "new test" do
    # expect(result).to eq(new stuff)
  end
end